### ISSUE TITLE
consistency_ice_veg_soil

### ISSUE THREAD

**Issue by rkutteh** [CABLE-LSM/CABLE#338]
Add a subroutine to CABLE to ensure the consistency of ice points between soil and vegetation as described in #280

**Comment by har917** [comment]
@ccarouge @rkutteh Noting that there is potential for issue creep here - could we extend this to also ensure consistency in other areas?  I'm particularly thinking ensure that lakes have 'soil' properties consistent with water and zero-vegetation.

**Comment by ccarouge** [comment]
@rkutteh Why do we need this issue? Isn't is a duplicate of #280?

**Comment by rkutteh** [comment]
@ccarouge please feel free to close #280 as the present issue is the one connected with my branch 338-consistency_ice_veg_soil and the benchcab testing I am doing now, and it supersedes #280

**Comment by ccarouge** [comment]
I'd rather not close #280 until it is fixed since it is the issue with all the discussions in. So I've connected to pull request to both issues and both will be closed once the pull request is merged.

In the future, please try not to duplicate issues. In this case, you should have linked the branch to the existing issue (create it from the existing issue page for example) instead of creating this issue to create (?) the branch afterwards.
